bvm/ballerina-rt/build.gradle

```diff
@@ -84,6 +84,7 @@ dependencies {
     dist 'org.slf4j:slf4j-api:1.7.22'
     dist 'com.moandjiezana.toml:toml4j:0.7.2'
```

Comment:
Can we remove the `com.moandjiezana.toml:toml4j:0.7.2` dependency with this change?